src/certgen/clu_certgen_ecc.c

```diff
@@ -1,53 +1,86 @@
-#include "clu_include/certgen/clu_certgen.h"
+/* clu_certgen_ecc.c
+ *
+ * Copyright (C) 2006-2020 wolfSSL Inc.
```

Comment:
Should we change 2020 to 2021?

Reply:
Yes, I can run the license script on the repo after.